fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DeleteResourcePersister.java

```diff
@@ -47,81 +42,28 @@ protected DeleteResourcePersister(final FedoraToOcflObjectIndex fedoraOcflIndex)
     public void persist(final OcflPersistentStorageSession session, final ResourceOperation operation)
             throws PersistentStorageException {
         final var mapping = getMapping(session.getId(), operation.getResourceId());
-        final var fedoraResourceRoot = mapping.getRootObjectIdentifier();
         final var resourceId = operation.getResourceId();
+
         final var objectSession = session.findOrCreateSession(mapping.getOcflObjectId());
-        final var user = operation.getUserPrincipal();
-        final var deleteTime = Instant.now();
-        log.debug("Deleting {} from {}", resourceId, mapping.getOcflObjectId());
-        if (fedoraResourceRoot.equals(resourceId)) {
-            // We are at the root of the object, so delete all the data files.
-            try {
-                if (objectSession.isNewInSession()) {
```

Comment:
All of the logic around handling staged or versioned files is now in fcrepo-storage-ocfl?

Reply:
Yep